src/eventsub/conduit/mod.rs

```diff
@@ -0,0 +1,8 @@
+#![doc()]
```

Comment:
this doc attribute is not needed

Reply:
removed (thought it would may be useful for consistency with the project structure)